File: src/main/java/co/com/fsighttech/imp/issuing/bo/configuration/ProductCoverage.java
/**
 * 
 */
package co.com.fsighttech.imp.issuing.bo.configuration;

import java.util.List;

import co.com.fsighttech.imp.issuing.bo.core.CoverageGroup;
import co.com.fsighttech.imp.issuing.bo.core.Product;
import lombok.Getter;
import lombok.Setter;

/**
 * This class represents the coverages associate to the product 
 * 
 * @author jtorres
 */
@Getter
@Setter
public class ProductCoverage {

	private Product product;
	private List<CoverageGroup> coveragesGroupList;
	
	
	
}

File: src/main/resources/static/src/app/agent-portal/shared/model/configuration/product-coverage.ts
import { Company } from '../core/company';
import { Product } from '../core/product';
import { CoverageGroup } from '../core/coverage-group';

/**
* this object represents the structure of coverage for a specific product
*
* @author jtorres
*/
export class ProductCoverage {
    product: Product;
    coveragesGroupList:CoverageGroup[];
}